tests/unit/test_utils.py

```diff
@@ -1,11 +1,93 @@
+import datetime
+
 import numpy as np
+import pandas as pd
 import pytest
 import xarray as xr
 from packaging.version import Version
 
 import mesmer.core.utils
 
 
+def make_dummy_yearly_data(freq):
+    if freq == "YM":
+        time = [datetime.datetime(i, 7, 15) for i in range(2000, 2005)]
+    else:
+        time = xr.cftime_range(start="2000", periods=5, freq=freq)
```

Comment:
```suggestion
        time = xr.time_range(start="2000", periods=5, freq=freq, calendar=calendar)
```

If you use `xr.time_range` you can also `parametrize` over different calendars (e.g. `("standard", "noleap")` )